app_api/routes/index.js

```diff
@@ -0,0 +1,20 @@
+var express = require('express');
+var router = express.Router();
+
+var ctrlStuffs = require('../controllers/stuffs');
```

Comment:
Looks good, I would still like to see 
- [ ] The creation of unit tests for the controllers of this routes
- [ ] Endpoint testing as well